### ISSUE TITLE
Reset timestamp faces when theme changes

### ISSUE THREAD

**Issue by treed** [alphapapa/ement.el#42]
The automatic generation of faces for the time column is pretty sweet, but if I toggle between light and dark themes, the faces don't get updated. I could presumably add something to `modus-themes-after-load-theme-hook` in my case, but I'd need a function to call which can do that.

For now the workaround is closing the rooms list buffer and reopening it, so it's not high priority, but I figured I'd put the request in.

**Comment by treed** [comment]
Looking at the code, this should be relatively simple to move out to a separate function. I might take a shot at it later.

**Comment by treed** [comment]
Yeah, I might refactor it also because I've been working on copying ement-room-list for an ement-device-list and I kinda want to do the same trick there, so maybe there could be generic timestamp faces that both lists could use and one function to generate them.

I've contacted the FSF and sent back their initial form and am waiting to hear back from that. I've verified with my company that they've got no issues with my contributing to FOSS in my free time so I don't anticipate any roadblocks once they come back to me with whatever forms.

**Comment by alphapapa** [comment]
@treed Time sure has flown.  :)  Any update on the FSF CA process?  (Or did you already do it? ISTR but forget...)  I'd still be glad for you to contribute this change.
